exercise4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.IO;
+using System.Xml.Linq;
+using Newtonsoft.Json;
+
+namespace exercise4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                var jsonFile = File.ReadAllText("E:\\Projects\\Quoc_Lab_5\\employees.json");
```

Comment:
- Don't put the absolute path.
- You can use `@` before the string instead of `\\`

Reply:
done